app-services/secrets/README.md

```diff
@@ -48,49 +124,21 @@ Success! Disabled the secrets engine (if it existed) at: secret/
 Success! Enabled the kv secrets engine at: secret/
 ```
 
-#### Insecure mode
-
-When security is disabled, the secrets can be written to and are retrieved from the *Writable.InsecureSecrets* section of the configuration file. Insecure secrets and their paths can be configured as below.
-
-```toml
- [Writable.InsecureSecrets]    
-      [Writable.InsecureSecrets.NoPath]
-        Path = ''
-        [Writable.InsecureSecrets.NoPath.Secrets]
-          username = 'nopath-user'
-          password = 'nopath-pw'
-      
-      [Writable.InsecureSecrets.AWS]
-        Path = 'aws'
-        [Writable.InsecureSecrets.AWS.Secrets]
-          username = 'aws-user'
-          password = 'aws-pw'
-      
-      [Writable.InsecureSecrets.MongoDB]
-        Path = 'mongodb'
-        [Writable.InsecureSecrets.MongoDB.Secrets]
-          username = ''
-          password = ''
-```
-
-`NOTE: An empty path is a valid configuration for a secret's location  `
-
-#### Run StoreSecrets and GetSecrets
-
-##### StoreSecrets
+5. Run the secrets service: `go run main.go`
+6. Follow the instructions in [Storing and Getting Secrets](#storing and getting secrets) in order to test storing and retrieving secrets from the secret store.
```

Comment:
links can't have spaces -- they don't render. Need to replace spaces with hyphens

Reply:
[Storing and Getting Secrets](#storing-and-getting-secrets)